wntr/epanet/util.py

```diff
@@ -571,7 +571,7 @@ def _to_si(self, flow_units, data, darcy_weisbach=False):
 
         elif self in [HydParam.RoughnessCoeff] and darcy_weisbach:
             if flow_units.is_traditional:
-                data = data * (1000.0 * 0.3048)  # 1e-3 ft to m
+                data = data * (0.001 * 0.3048)  # 1e-3 ft to m
```

Comment:
I'm not sure this is right. If the data is coming in from a "traditional" unit category and we are using DW, then the headloss units should be in thousandth of a foot (ie 0.001 ft). To convert that to mm I do the following dimensional analysis:

1 (e-3 ft) * (1 ft / 1000 e-3 ft) * ( 304.8 mm / 1 ft) = (304.8/1000) mm

So the code should be `data = data * (0.3048 / 1000.0)`.